include/rocksdb/db.h

```diff
@@ -480,6 +480,25 @@ class DB {
         keys, values);
   }
 
+  virtual std::vector<Status> MultiGet(
+      const ReadOptions& /*options*/,
+      const std::vector<ColumnFamilyHandle*>& /*column_family*/,
+      const std::vector<Slice>& keys, std::vector<std::string>* /*values*/,
+      std::vector<std::string>* /*timestamps*/) {
+    return std::vector<Status>(
```

Comment:
Any particular reason this API is not supported?

Reply:
It's one of the basic methods that cannot be implemented by calling other methods on the interface. I could have made it pure virtual, then all the classes (quite a few) deriving from this interface (e.g. stackables, transaction dbs) need to be touched to provide an (dummy) implementation. DBImpl does implement this method. Those derived classes can override and provide their implementation if so desired.

Reply:
SG.